website/src/views/conceptMap.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
 
 export default function ConceptMap() {
+
     return (
-        <div>
+        <div style={{ textAlign: 'center' }}>
             <h1>Concept Map</h1>
+            <iframe className="concept_map_iframe" id="ConceptMap" title="Concept Map"
+                src="http://localhost:8080?show_legend=false" 
```

Comment:
This should be a container name

Reply:
Resolved in [c67e6c0](https://github.com/Connor-Bernard/gradeView/pull/96/commits/c67e6c0ec1c1415fbd8a9084e7c270b322e31045)